mkdocs.yml

```diff
@@ -12,10 +12,11 @@ extra_javascript: [js/vendor/jquery-3.1.0.min.js, js/yaml.js]
 
 pages:
     - Home: 'index.md'
+    - Cluster Management:
+        - Set Up a Screwdriver Cluster: 'cluster-management/cluster-setup.md'
```

Comment:
Isn't this only for running in Kubernetes?  Should we call this something related to that?